doc/perception_configuration/perception_configuration_tutorial.rst

```diff
@@ -19,11 +19,13 @@ We will have to generate a YAML configuration file for configuring the 3D sensor
      point_subsample: 1
      padding_offset: 0.1
      padding_scale: 1.0
+     max_update_rate: 1.0
      filtered_cloud_topic: filtered_cloud
 
 **The general parameters are:**
 
 * *sensor_plugin*: The name of the plugin that we are using.
+* *max_update_rate*: The rate at which the octomap representation will be updated.
```

Comment:
This needs to be re-worded. This says that the octomap will update 'at' rather than 'up to' this value